talk/setup.tex

```diff
@@ -100,7 +101,7 @@
 %%%%%%%%%%%%%%%%%%%
 % useful commands %
 %%%%%%%%%%%%%%%%%%%
-\newcommand{\cpp}{C$^{++}$}
+\newcommand{\cpp}{C$^{++}$\xspace}
```

Comment:
Yeah, I played with `\xspace` as well, but I was too afraid to put it in the course slides yet :)